packages/dotcom-ui-header/src/header.scss

```diff
@@ -67,5 +67,34 @@
 }
 
 .ft-header__drawer-ask-ft-button {
-  margin-left: 16px;
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  text-align: center;
+
+}
+
+.edition-switcher {
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  padding-left: 16px;
+  font-family: MetricWeb;
+
+  .o-header__drawer-menu-link {
+    padding-top: 6px;
+    padding-bottom: 6px;
+    font-size: 16px;
```

Comment:
You don't need to set these declarations (font-size & font-family) again. They should propagate from parent to child in this case _(if they needed to be set at all)_

Reply:
i need to set it because o-header__drawer-menu-link have this own font size setted that is 18px, im using css specificity to change the font size of the link inside the edition-switcher. 
the font size of the parent has less specificity than the own style of the class